dotnet/src/Functions/Functions.OpenAPI/OpenApi/OpenApiDocumentParser.cs

```diff
@@ -285,6 +286,74 @@ private static List<RestApiOperationParameter> CreateRestApiOperationParameters(
         return new RestApiOperationPayload(mediaType, payloadProperties, requestBody.Description);
     }
 
+    private static RestApiOperationResponse? CreateRestApiOperationResponses(OpenApiResponses responses)
+    {
+        if (responses == null)
+        {
+            return null;
+        }
+
+        // TODO: support default/no response code
+        // TODO: support other response codes
+        var response = responses.FirstOrDefault(r => r.Key == "200");
+
+        if (response.Value == null)
+        {
+            return null;
+        }
+
+        var mediaType = s_supportedMediaTypes.FirstOrDefault(smt => response.Value.Content.ContainsKey(smt));
+        if (mediaType == null)
+        {
+            // TODO -- use text/plain or application/json as default?
+            return new RestApiOperationResponse(string.Empty, string.Empty);
+        }
+
+        var mediaTypeMetadata = response.Value.Content[mediaType];
+
+        // Parse schema
+        var schema = ParseSchema(mediaTypeMetadata.Schema);
+
+        // TODO - Kind of overloading `content` and `contentType` properties here.
+        return new RestApiOperationResponse(mediaType, response.Value.Description ?? mediaTypeMetadata.Schema?.Description ?? string.Empty, schema);
+    }
+
+    private static RestApiOperationResponseSchema? ParseSchema(OpenApiSchema schema)
+    {
+        if (schema == null)
+        {
+            return null;
+        }
+
+        var apiResponseSchema = new RestApiOperationResponseSchema
+        {
+            Title = schema.Title,
+            Type = schema.Type
+            // Add other properties as needed
+        };
+
+        if (schema.Type == "object" && schema.Properties != null)
+        {
+            foreach (var property in schema.Properties)
+            {
+                // Recursively parse nested schemas
+                var parsedSchema = ParseSchema(property.Value);
```

Comment:
@alliscode Here is where I did some recursion to get things working for me. Maybe similar is needed for the `JsonSchema.Net` stuff.